<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mat3DTransformView</class>
 <widget class="QWidget" name="Mat3DTransformView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="position">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="posX">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="posY">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="posZ">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="rotation">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="rotX">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="rotZ">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="rotY">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="scale">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="scaleX">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="scaleZ">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="scaleY">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
